curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-mixed-grouping-of-characters.english.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 5c3dda8b4d8df89bea71600f
+title: Check For Mixed Grouping of Characters
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+Sometimes we want to check for groups of characters using a Regular Expression, to achieve this task we use the round brackets <code>()</code>.
+If you want to find either <code>"Penguin"</code> or <code>"Pumpkin"</code> in a string, you could use the following Regular Expression: <code>/P(engu|umpk)in/g</code>
+Then you can test the code using the <code>test()</code> method to find whether the requires character(s)/string(s) are in the test string or not.
+<blockquote>let testStr = "Pumpkin";<br>let testRegex = /P(engu|umpk)in/g;<br>testRegex.test(testStr);<br>// Returns true</blockquote>
+</section>
+
+## Instructions
+<section id='instructions'>
+Fix the RegEx so that it checks for either <code>Hello</code> or <code>Halo</code> in a case sensitive manner.
```

Comment:
RegEx: pick one spelling and stick to it. The most common in this exercise has been without abbreviation, but do check what the ones before and after use.

Reply:
I'll change everything to regex, seems like the most used in the curriculum